### PR TITLE
CIS v1.11 User documentation Updates

### PR DESCRIPTION
**Pull request by snatra**
Added CIS rel v1.11 documentation

Primary reviewers:
@mdditt2000 @kkinch @ssurenr

### CODE REVIEW

File: docs/openshift/kctlr-use-as3-backend.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ Prerequisites
 
 Ensure you meet the following requirements:
 
-- BIG-IP Controller v1.10 or higher when used in OpenShift.
+- BIG-IP Controller v1.11 or higher when used in OpenShift.
```
Comment (ssurenr) [inline docs/openshift/kctlr-use-as3-backend.rst]:
```suggestion
- BIG-IP Controller v1.10 or higher when used in OpenShift.
```
This feature is introduced in 1.10. Minimum requirements should be kept at that version.

File: docs/openshift/kctlr-use-as3-backend.rst [hunk 2]
```diff
@@ -88,11 +88,12 @@ Annotations
 - virtual-server.f5.com/clientssl
 - virtual-server.f5.com/serverssl
 - virtual-server.f5.com/secure-serverssl
+- virtual-server.f5.com/waf
 
 Known Issues
 ------------
 
-v1.10.0
+v1.11.0
```
Comment (ssurenr) [inline docs/openshift/kctlr-use-as3-backend.rst]: Add a new section v1.11.0 above the v1.10.0 section and add limitations specific to v1.11.0 release that are NOT part previous releases.

File: docs/openshift/kctlr-use-as3-backend.rst (hunk 1 above)
Comment (snatra) [reply docs/openshift/kctlr-use-as3-backend.rst]: Ok Sure

File: docs/openshift/kctlr-use-as3-backend.rst (hunk 2 above)
Comment (snatra) [reply docs/openshift/kctlr-use-as3-backend.rst]: Ok will make changes accordingly

**Pull request merged**